modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptAxiosClientCodegen.java

```diff
@@ -127,6 +131,15 @@ public void processOpts() {
             }
         }
 
+        if (additionalProperties.containsKey(STRING_ENUMS)) {
+            this.stringEnums = Boolean.parseBoolean(additionalProperties.get(STRING_ENUMS).toString());
+            additionalProperties.put("stringEnums", this.stringEnums);
+            if (this.stringEnums) {
+                enumSuffix = "";
```

Comment:
Why is this needed?

Reply:
There is no particular reason.
I reused code from [here](https://github.com/OpenAPITools/openapi-generator/blob/b7c8de94250905c24b4b386a1054a3b5e8bb6537/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptNestjsClientCodegen.java#L145-L152).

Reply:
I’ll remove this code for the following reasons:
- Users may be confused if the name of the enum changes depending on the options.
- In this generator, `classEnumSeparator` is not used.